### PR TITLE
FIX: unnecessary rawsource

### PR DESCRIPTION
**Pull request by drafish**
fix issue #623

### CODE REVIEW

**Comment by chrisjsewell** [comment]
Thanks @drafish I will try to have a look at this soon; its just not clear to me if this is a "complete" solution for dealing with paragraph raw text.
Also obviously the test need to be fixed (hint you can run `pytest --force-regen` to auto-update https://pytest-regressions.readthedocs.io/ fixtures)

Perhaps also @jpmckinney wants to have a look, as the resident translation expert 😄

File: myst_parser/mdit_to_docutils/base.py [hunk 1]
```diff
@@ -439,7 +439,13 @@ def renderInlineAsText(self, tokens: list[SyntaxTreeNode]) -> str:
     # ### render methods for commonmark tokens
 
     def render_paragraph(self, token: SyntaxTreeNode) -> None:
-        para = nodes.paragraph(token.children[0].content if token.children else "")
+        if token.children:
+            if token.children[0].children and len(token.children[0].children) == 1:
```
Comment (jpmckinney) [inline myst_parser/mdit_to_docutils/base.py:443]: Can you add a test case for this scenario? (the image markup example from the issue description)

Comment (jpmckinney) [reply myst_parser/mdit_to_docutils/base.py:443]: I'm also not sure that this is the correct condition. I assume it works to correct the specific issue with the image markup, but wouldn't this also mean that any paragraph with a single child node ends up with no content?

**Comment by drafish** [comment]
I change my mind. I always thought image does not need to be translated. But that's not right. Not only alt can be translated, image can also be translated. MyST is right, recommonmark is wrong.

Even if my thought is right, my code is wrong. I may have tested it in a wrong way. Now I tested it again, my code can not render image correctly. 

I am sorry for my careless work. I should trust MyST more. I am closing this pr now. @chrisjsewell @jpmckinney

**Pull request closed without merging**